Gemfile

```diff
@@ -98,6 +98,8 @@ gem 'httparty'
 gem 'icalendar', require: false
 # JSON Web Token encoding / decoding (e.g. for links in e-mails)
 gem "jwt"
+# Tool to detect unused code through knowing which methods are used in which files.
+gem 'coverband'
```

Comment:
Should we configure view tracking? https://github.com/danmayer/coverband#view-tracking

Reply:
Ahh yes! I thought it was by default added.